dbt/models/reporting/schema.yml

```diff
@@ -98,6 +98,28 @@ models:
           - property_group
           - assessment_stage
 
+  - name: reporting.vw_assessment_progress
```

Comment:
**suggestion (non-blocking):** It might be worth materializing this view by adding the dbt `daily` tag, as it queries ~50GB of data for each run.